ivadomed/loader/film.py

```diff
@@ -274,12 +274,12 @@ def save_film_params(gammas, betas, metadata_values, depth, ofolder):
 
     # Save the numpy arrays for gammas/betas inside files.npy in log_directory
     for i in range(1, 2 * depth + 3):
-        gamma_layer_path = os.path.join(ofolder, "gamma_layer_{}.npy".format(i))
-        np.save(gamma_layer_path, gammas_dict[i])
-        beta_layer_path = os.path.join(ofolder, "beta_layer_{}.npy".format(i))
-        np.save(beta_layer_path, betas_dict[i])
+        gamma_layer_path = Path(ofolder).joinpath("gamma_layer_{}.npy".format(i))
+        np.save(str(gamma_layer_path), gammas_dict[i])
+        beta_layer_path = Path(ofolder).joinpath("beta_layer_{}.npy".format(i))
```

Comment:
Instead of `.format()`, let's use f string interpolation for better readability.

Also instead of `joinpath` use `/` operator for path as that is the primary advantage of adopting path format.

Reply:
> Instead of `.format()`, let's use f string interpolation for better readability.
> 
> Also instead of `joinpath` use `/` operator for path as that is the primary advantage of adopting path format.

i think that instead of using joinpath a better way is just to use Path(a, b, c, d), this way it accepts both when the parameters is a string or the parameter is a path